docs/Guides/Serverless.md

```diff
@@ -22,6 +22,17 @@ application directly without the need for any additional tools, while the same
 code will be executed in your serverless platform of choice with an additional
 snippet of code.
 
+**Note**:
+
+Fastify will not allow adding more routes or registering plugins once `ready()` is called because [Fastify will "compile" all of your plugins and routes](https://github.com/fastify/fastify/issues/1771#issuecomment-515339364).
```

Comment:
```suggestion
Fastify will not allow adding more routes or registering plugins once `ready()` 
is called because [Fastify will freeze all of your plugins and routes](https://github.com/fastify/fastify/issues/1771#issuecomment-515339364)
into a fast and immutable data structure.
```